### PR TITLE
Video Component suggestions...

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: src/app/index.html [hunk 1]
```diff
@@ -44,7 +44,15 @@
   </stache-page-anchor>
 
   <stache-image
-    imageSource="https://placecage.com/200/200">
+    imageSource="https://unsplash.it/200/300">
```
Comment (Blackbaud-SteveBrush) [inline src/app/index.html]: Need SSL support, plus this is boring and plain :)

File: src/app/public/src/modules/video/video.component.scss [hunk 2]
```diff
@@ -3,19 +3,30 @@
 .stache-video {
   position: relative;
   display: block;
-  height: 0;
-  padding: 0;
-  padding-bottom: 56.25%;
+  width: 100%;
   overflow: hidden;
-  margin: 2em 0;
+  margin-top: 1em;
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/src/modules/video/video.component.scss]: `1em` is what's generally used for block elements throughout the app.

File: src/app/index.html (hunk 1 above)
Comment (blackbaud-brandonhare) [inline src/app/index.html]: LAAAAAME.

File: src/app/public/src/modules/video/video.component.scss (hunk 2 above)
Comment (blackbaud-brandonhare) [inline src/app/public/src/modules/video/video.component.scss]: Yeah, I was gonna change that after the Image PR. Makes sense to be consistent.

**Pull request merged**